pkg/sql/schemachanger/scexec/scmutationexec/index.go

```diff
@@ -466,6 +474,14 @@ func (m *visitor) RemoveColumnFromIndex(ctx context.Context, op scop.RemoveColum
 			idx.StoreColumnIDs = idx.StoreColumnIDs[:i]
 		}
 	}
+	// Only the last column can be inverted, so there is only one spot
+	// to unset this.
+	switch op.InvertedKind {
+	case scpb.IndexColumn_INVERTED:
+		fallthrough
+	case scpb.IndexColumn_INVERTED_TRIGRAM:
```

Comment:
nit `case scpb.IndexColumn_INVERTED, scpb.IndexColumn_INVERTED_TRIGRAM:` and then you don't need to `fallthrough`